keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c

```diff
@@ -0,0 +1,295 @@
+/* Copyright 2021 Kyle McCreery
+ * Copyright 2021 Jonavin Eng
+ * Copyright 2022 Fearless Spiff
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+#include <stdio.h>
+#include "print.h"
```

Comment:
```suggestion
```

Reply:
And thanks again! Done also with abeb90b